todo-app/src/App.js

```diff
@@ -1,13 +1,18 @@
-import React, { Component } from 'react';
+//jshint esnext: true
 
-class App extends Component {
+import React from 'react';
+
+import Todos from './Components/todoList';
```

Comment:
I think we had a miscommunication on last week's homework about naming conventions. React component files are normally named exactly as the component is named, so in this case it would be `TodoList`. The uppercase first letter tells us immediately that it's a component.

Also, if you are calling your component `TodoList` via naming the file, and also naming the component inside the file, you should call it the exact same when you import it, eg:

    import TodoList from './Components/TodoList';